packages/kbn-search-errors/src/es_error.tsx

```diff
@@ -9,22 +9,32 @@
 import React from 'react';
 import { EuiButton, EuiCodeBlock, EuiSpacer } from '@elastic/eui';
 import { i18n } from '@kbn/i18n';
-import { ApplicationStart } from '@kbn/core/public';
-import { KbnError } from '@kbn/kibana-utils-plugin/common';
+import type { ApplicationStart } from '@kbn/core/public';
 import { IEsError } from './types';
 import { getRootCause } from './utils';
 
-export class EsError extends KbnError {
+/**
+ * Checks if a given errors originated from Elasticsearch.
+ * Those params are assigned to the attributes property of an error.
+ *
+ * @param e
+ */
+export function isEsError(e: any): e is IEsError {
+  return !!e.attributes;
+}
+
+export class EsError extends Error {
```

Comment:
Does it matter that this no longer extends `KbnError`?

Reply:
had to revert change so instanceof checks worked fro extended classes like PainlessError